assets/js/base/context/cart-checkout/checkout-state/index.js

```diff
@@ -135,22 +134,25 @@ export const CheckoutStateProvider = ( {
 
 	// emit events.
 	useEffect( () => {
-		const status = checkoutState.status;
+		const { hasError, status } = checkoutState;
 		if ( status === STATUS.PROCESSING ) {
-			emitEventWithAbort(
+			emitEvent(
```

Comment:
If we go with these changes, `emitEventWithAbort` won't be used anywhere, so we could remove that method if we don't see a usage in the near future.